base/applications/cmdutils/mode/lang/zh-CN.rc

```diff
@@ -59,14 +61,14 @@ BEGIN
     IDS_ERROR_SET_PARALLEL_STATE "错误: SetParallelState(%d) - DefineDosDevice(%s) (0x%lx)\n"
     IDS_ERROR_INVALID_PARAMETER "无效参数 - %s\n"
     IDS_ERROR_ILLEGAL_DEVICE_NAME "非法设备名 - %s (0x%lx)\n"
-    IDS_ERROR_DEVICE_NOT_AVAILABLE "The device %s is not currently available.\n"
-    IDS_ERROR_STATUS_GET_DEVICE "无法获得设备 %s: 的状态\n"
-    IDS_ERROR_STATUS_SET_DEVICE "无法设置设备 %s: 的状态\n"
-    IDS_ERROR_TIMEOUT_GET_DEVICE "Failed to get the timeout for device %s:\n"
-    IDS_ERROR_TIMEOUT_SET_DEVICE "Failed to set the timeout for device %s:\n"
+    IDS_ERROR_DEVICE_NOT_AVAILABLE "此设备目前不可用。\n"
```

Comment:
> Disclaimer: I am not Simplified Chinese user.

Possible to use "当前" here? Because "目前" seems rarely used in Simplified Chinese, especially in Mainland China.

Reply:
Yes, "当前" one seems More appropriate.